events/permissions.py

```diff
@@ -180,7 +181,10 @@ def has_object_permission(self, request: Request, view, obj):
         if (
             settings.WEB_STORE_INTEGRATION_ENABLED
             and request.method in ("PUT", "PATCH")
-            and request.data.get("web_store_merchants")
+            and (
+                request.data.get("web_store_merchants")
+                or request.data.get("web_store_accounts")
+            )
```

Comment:
Not really related to this PR, but user who has both admin and financial admin permissions should be able to update all fields of the organization